frontend/src/modules/modals/MultiStepWizard.tsx

```diff
@@ -405,16 +294,67 @@ const MultiStepWizard = (): JSX.Element => {
   /**
    * State to manage Parameter `Data`
    */
-  const [parameter, setParameter] = React.useState<Data>();
+  const [parameter, setParameter] = React.useState<InternalState>();
   /**
    * State to manage all Parameter `Data`
    */
-  const [multiParams, setMultiParams] = React.useState<Data[]>([]);
+  const [multiParams, setMultiParams] = React.useState<InternalState[]>([]);
   /**
    * State to manage if button submitted is disabled
    */
   const [isSubmitDisabled, setIsSubmitDisabled] = React.useState(false);
 
+  const alarmValuesSpO2 = useSelector(getSpO2AlarmLimitsRequest, shallowEqual);
+  const alarmValuesHR = useSelector(getHRAlarmLimitsRequest, shallowEqual);
```

Comment:
Whenever this file calls something `alarmValues`, does it actually mean `alarmLimits`?